package.json

```diff
@@ -5,8 +5,7 @@
     "browserslist": "> 0.25%, not dead",
     "scripts": {
         "build": "webpack",
-        "start": "webpack-dev-server",
-        "dev": "npm run start",
```

Comment:
I don't see any reason to remove the duplicate script. Our guys are used to running npm run dev since that is what the command initially was.